pandas/core/frame.pyi

```diff
@@ -57,7 +57,7 @@ class _iLocIndexerFrame(_iLocIndexer):
 
 class _LocIndexerFrame(_LocIndexer):
     @overload
-    def __getitem__(self, idx: Union[Union[IndexType, MaskType], List[StrLike], Tuple[Union[IndexType, MaskType, slice, List[StrLike]], Union[List[StrLike], slice, Series[bool]]]],) -> DataFrame: ...
+    def __getitem__(self, idx: Union[Union[IndexType, MaskType], List[StrLike], Tuple[Union[IndexType, MaskType, slice, List[StrLike], ...], Union[List[StrLike], slice, Series[bool]]]],) -> DataFrame: ...
```

Comment:
This doesn't seem to be right:

![image](https://user-images.githubusercontent.com/5341706/136612445-8fcf0b3d-bc02-4764-8212-3ef4cd78fa2b.png)

The brackets here imply that the tuple is this:

```
Tuple[Union[IndexType, MaskType, slice, List[StrLike], ...], Union[List[StrLike], slice, Series[bool]]]
```

Reply:
Whereas the `...` is being added to the `Union`.

Reply:
Ooops.  I fixed it in commit https://github.com/microsoft/python-type-stubs/pull/103/commits/c851bcc1ee0768c217acbdd998079a283a25713c